app/services/ror_service.rb

```diff
@@ -26,7 +26,7 @@ def call
   def organizations
     conn.get('/v2/organizations', params, headers).body
   rescue Faraday::Error => e
-    raise Error, "Connection err: #{e.message}"
+    raise StandardError, "RoR Service connection error: #{e.message}"
```

Comment:
If we're not going to handle this error in any way, then there is no reason to rescue the Faraday::Error at all.

Reply:
Yup.